modules/apps/polls/polls-web/src/main/java/com/liferay/polls/web/internal/portlet/display/context/PollsDisplayContext.java

```diff
@@ -56,10 +56,12 @@
 public class PollsDisplayContext {
 
 	public PollsDisplayContext(
-		RenderRequest renderRequest, RenderResponse renderResponse) {
+		RenderRequest renderRequest, RenderResponse renderResponse,
+		PollsQuestionLocalService pollsQuestionLocalService) {
```

Comment:
@carolmariaabb, do we need to pass `pollsQuestionLocalService` as parameter or we can just inject its dependency on `PollsDisplayContext`?

Reply:
I believe it's possible to inject its dependency on PollsDisplayContext, but I thought it would be better to follow the standard from others portlets:

DDMFormAdminFieldSetDisplayContext
DDMFormDisplayContext
DDMDataProviderDisplayContext
DDMFormViewFormInstanceRecordDisplayContext
DDMFormViewFormInstanceRecordsDisplayContext
DDMFormBrowserDisplayContext

Reply:
I believe it's possible but I thought it would be better to follow the pattern from other portlets:
```
DDMFormAdminFieldSetDisplayContext
DDMFormDisplayContext
DDMDataProviderDisplayContext
DDMFormViewFormInstanceRecordDisplayContext
DDMFormViewFormInstanceRecordsDisplayContext
DDMFormBrowserDisplayContext
DDLDisplayContext
```
I also checked on others and didn't see any injecting into the display context.

Reply:
Nice job, @carolmariaabb! 👏 Keep following the existent code pattern as you did here. 😉

Reply:
Thanks!